### PR TITLE
[DOCS-5990] Update Full Text Search for negation

### PR DESCRIPTION
**Pull request by Prosune**

### CODE REVIEW

File: search-services/1.3/using/index.md [hunk 1]
```diff
@@ -289,13 +289,14 @@ Single terms, phrases, and so on can be combined using "`NOT`" in upper, lower,
...
-yellow !banana
-yellow -banana
+yellow AND NOT banana
+yellow AND !banana
+yellow AND -banana
 NOT yellow banana
 -yellow banana
 !yellow banana
 ```
+The `NOT` operator can be omitted for Share.
```
Comment (tpage-alfresco) [inline search-services/1.3/using/index.md:299]: I don't understand how this can be the case.

Comment (Prosune) [reply search-services/1.3/using/index.md:299]: @tpage-alfresco I checked with @mjstanford. He tested the change and it looks good.

Comment (tpage-alfresco) [reply search-services/1.3/using/index.md:299]: To be clearer - I don't see how it makes sense for these queries to return the same results from Share:
```
yellow banana
NOT yellow banana
```

Comment (mjstanford) [reply search-services/1.3/using/index.md:299]: I see your point now. Agree. I was focused on the change Prosune is making (which is correct), but the docs were wrong before. "yellow banana" and "NOT yellow banana" will never return the same results, which is implied by "These queries search for nodes that contain the terms `yellow` in any content."

Suggestions to be added in review comments.

Comment (mjstanford) [inline search-services/1.3/using/index.md:299]: Incorrect. The "AND" can be omitted for Share. Not the "Not".

File: search-services/1.3/using/index.md [hunk 2]
```diff
@@ -289,13 +289,14 @@ Single terms, phrases, and so on can be combined using "`NOT`" in upper, lower,
 These queries search for nodes that contain the terms `yellow` in any content.
 
 ```sql
-yellow NOT banana
-yellow !banana
-yellow -banana
+yellow AND NOT banana
+yellow AND !banana
+yellow AND -banana
 NOT yellow banana
```
Comment (mjstanford) [inline search-services/1.3/using/index.md:295]: Add line here "These queries search for nodes that contain the term `banana` but do not contain "yellow" in any content."

File: search-services/1.3/using/index.md [hunk 3]
```diff
@@ -289,13 +289,14 @@ Single terms, phrases, and so on can be combined using "`NOT`" in upper, lower,
 These queries search for nodes that contain the terms `yellow` in any content.
```
Comment (mjstanford) [inline search-services/1.3/using/index.md:289]: Update to say ""These queries search for nodes that contain the term "yellow" but do not contain "banana" in any content."

**Review by mjstanford — CHANGES_REQUESTED** [review]

File: search-services/1.3/using/index.md (hunk 1 above)
Comment (Prosune) [reply search-services/1.3/using/index.md:299]: So, just to make myself clear, the meaning of the following sequence is as follows:
```yellow AND NOT banana
yellow AND !banana
yellow AND -banana
NOT yellow banana
-yellow banana
!yellow banana```

In the first 3 lines, "yellow" is being searched and "banana" is omitted. In the next 3 lines "banana" is being searched and "yellow" is omitted. 

Also, is there an instance where `yellow banana` and `NOT yellow banana` are mentioned in the same example set? I did not find one.

Comment (Prosune) [reply search-services/1.3/using/index.md]: @tpage-alfresco @mjstanford I have updated the suggested changes. Let me know if it looks good now or whether I have missed something.

File: search-services/1.3/using/index.md [hunk 4]
```diff
@@ -286,16 +286,21 @@ You can narrow your search results by excluding words with the `NOT` syntax.
 
 Single terms, phrases, and so on can be combined using "`NOT`" in upper, lower, or mixed case, or prefixed with "`!`" or "`-`".
 
-These queries search for nodes that contain the terms `yellow` in any content.
+These queries search for nodes that contain the term "yellow" but does not contain the term "banana" in the content.
```
Comment (mjstanford) [inline search-services/1.3/using/index.md:289]: Minor correction needed. Change: 

"These queries search for nodes that contain the term "yellow" but do**es** not contain the term "banana" in the content."
To
"These queries search for nodes that contain the term "yellow" but do not contain the term "banana" in the content."

Swap "does" for "do". This is better English. Please apply to other similar lines on this page.

**Review by mjstanford — CHANGES_REQUESTED** [review]

File: search-services/1.3/using/index.md (hunk 4 above)
Comment (Prosune) [reply search-services/1.3/using/index.md]: Good catch! Thanks for the review.

File: search-services/1.3/using/index.md (hunk 3 above)
Comment (Prosune) [reply search-services/1.3/using/index.md]: Updated.

**Review by mbujak-hyland — APPROVED** [review]
LGTM

**Review by mjstanford — APPROVED** [review]

**Pull request merged**